orchestrator.Jenkinsfile

```diff
@@ -82,6 +82,18 @@ stage('build projects') {
     parallel jobs
 }
 
+if (runTests) {
+    stage('run tests for the examples') {
+        build job: 'examples/tests/examples',
+            wait: false,
```

Comment:
Is it intended not to wait for this job to finish?

Reply:
Yes, because that pipeline is only building the image right now. Wait will be switched on with a future PR when the deployment is also enabled.